cas/remote.go

```diff
@@ -17,24 +17,35 @@ package cas
 import (
 	"encoding/json"
 	"fmt"
+	"io"
+	"io/ioutil"
 	"net/http"
 	"os"
 	"time"
 
+	"github.com/coreos/rocket/pkg/keystore"
+
+	"github.com/appc/spec/aci"
 	"github.com/appc/spec/schema/types"
 	"github.com/coreos/rocket/Godeps/_workspace/src/github.com/mitchellh/ioprogress"
+	"github.com/coreos/rocket/Godeps/_workspace/src/golang.org/x/crypto/openpgp"
 )
 
-func NewRemote(name string, mirrors []string) *Remote {
```

Comment:
Can you please just scrap the mirrors/Mirrors entirely? It is highly misleading and pointless right now, we can reintroduce it when we actually create that functionality.